lib/node_modules/@stdlib/ndarray/base/count-falsy/test/test.1d.js

```diff
@@ -0,0 +1,88 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var toAccessorArray = require( '@stdlib/array/base/to-accessor-array' );
+var zeros = require( '@stdlib/array/zeros' );
+var ones = require( '@stdlib/array/ones' );
+var ndarray = require( '@stdlib/ndarray/ctor' );
+var countFalsy = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof countFalsy, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function counts the number of falsy elements in a 1-dimensional ndarray', function test( t ) {
+	var actual;
+	var x;
+
+	x = ndarray( 'float64', zeros( 8, 'float64' ), [ 4 ], [ 2 ], 1, 'row-major' );
+
+	actual = countFalsy( [ x ] );
+	t.strictEqual( actual, 4, 'returns expected value' );
+
+	x = ndarray( 'float64', ones( 8, 'float64' ), [ 4 ], [ 2 ], 1, 'row-major' );
+
+	actual = countFalsy( [ x ] );
+	t.strictEqual( actual, 0, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'the function counts the number of falsy elements in a 1-dimensional ndarray (accessors)', function test( t ) {
+	var actual;
+	var x;
+
+	x = ndarray( 'float64', toAccessorArray( zeros( 8, 'float64' ) ), [ 4 ], [ 2 ], 1, 'row-major' );
+
+	actual = countFalsy( [ x ] );
+	t.strictEqual( actual, 4, 'returns expected value' );
+
+	x = ndarray( 'float64', toAccessorArray( ones( 8, 'float64' ) ), [ 4 ], [ 2 ], 1, 'row-major' );
```

Comment:
```suggestion
	x = ndarray( 'generic', toAccessorArray( zeros( 8, 'generic' ) ), [ 4 ], [ 2 ], 1, 'row-major' );

	actual = countFalsy( [ x ] );
	t.strictEqual( actual, 4, 'returns expected value' );

	x = ndarray( 'generic', toAccessorArray( ones( 8, 'generic' ) ), [ 4 ], [ 2 ], 1, 'row-major' );
```

@headlessNode This applies throughout this PR (and likely others). When creating ndarrays backed by "generic" accessor arrays, the dtype should be `generic`, not `float64`. It doesn't matter that the true underlying buffer is `Float64Array`. From the perspective of the `ndarray`, the buffer is "generic", as `toAccessorArray` provides an abstracted interface over any collection.